lp2p/client/doc.go

```diff
@@ -0,0 +1,79 @@
+/*
+
+Package client provides a drand client implementation that retrieves
+randomness by subscribing to a libp2p pubsub topic.
+
+WARNING: this client can only be used to `Watch` for new randomness rounds and
+`Get` randomness rounds it has previously seen that are still in the cache.
+
+If you need to `Get` arbitrary rounds from the chain then you must combine this client with the http or grpc clients.
+
+The agnostic client builder must receive `WithChainInfo()` in order for it to validate randomness rounds it receives, or `WithChainHash()` and be combined with the http or grpc client implementations so that chain information can be fetched from them.
```

Comment:
```suggestion
The agnostic client builder must receive `WithChainInfo()` in order for it to validate randomness rounds it receives,
or `WithChainHash()` and be combined with the HTTP or gRPC client implementations so that chain
information can be fetched from them.
```

Reply:
changing this manually as I forgot to replace backquotes in this file for normal quotes too..